### ISSUE TITLE
Lack of Comments :D 

### ISSUE THREAD

**Issue by Ycy97** [aymansalama/wikidownloader_java#10]
Hi , if possible can add more comments to all the files so that we can understand it too :D thanks

**Comment by aymansalama** [comment]
I agree about the size of the comments. the best practice is 50% of the code is comments for documentation.
